src/main/java/com/mathworks/ci/MatlabBuilder.java

```diff
@@ -410,15 +413,16 @@ private synchronized int execMatlabCommand(Run<?, ?> build, FilePath workspace,
             throws IOException, InterruptedException {
         setEnv(build.getEnvironment(listener));
```

Comment:
this goes closer to the creation of the ProcLauncher

Reply:
in order to keep the execMatlab command parameters minimalistic, I have moved this up in the perform method.